pygaggle/rerank/transformer.py

```diff
@@ -247,3 +249,29 @@ def rescore(self, query: Query, texts: List[Text]) -> List[Text]:
             text.score = max(smax_val.item(), emax_val.item())
 
         return texts
+
+
+class CrossEncoderReranker(Reranker):
```

Comment:
I think calling this a CrossEncoderReranker while the others not is not a good naming convention since all of them are, how about change it to be SentenceTransformersReranker

Reply:
Sure, that'd be better!